src/bournemouth/chat_utils.py

```
@@ -0,0 +1,91 @@
+            if choice.finish_reason is not None:
+                async with send_lock:
+                    raw = encoder.encode(
+                        ChatWsResponse(
+                            transaction_id=transaction_id,
+                            fragment="",
+                            finished=True,
+                        )
+                    )
+                    await ws.send_text(raw.decode())
+                break
+    except (falcon.HTTPGatewayTimeout, falcon.HTTPBadGateway) as exc:  # pragma: no cover - passthrough
```

Ruff (0.11.9)

Line too long (103 > 88)

(E501)